### DISCUSSION TITLE
Can't access Vercel.com: "Application error: a client-side exception has occurred (see the browser console for more information)."

### DISCUSSION THREAD

**Discussion by exitcas** [vercel/community#1071] [Help]
Hi, I've been having this problem opening the app. Everything loads all right, but when you are about to do something, the screen flashes out to a white background and the legend "Application error: a client-side exception has occurred (see the browser console for more information)." written down at the middle of the website.

Someone else is getting out this type of errors? What could be causing this?

**Comment by swarnava** [comment]
Can you send a screencast?

**Reply by exitcas** [reply]
## Website

![imagen](https://user-images.githubusercontent.com/83518257/202910914-0b5db9d6-c3f5-440f-bceb-3672e62a3024.png)

## Logs

This logs feature some translations taking the form of little comments. This is because my browser is configured in Spanish.

![imagen](https://user-images.githubusercontent.com/83518257/202909793-ca1b3d68-d66a-40f6-8fa8-e35f04ac3c8b.png)
![imagen](https://user-images.githubusercontent.com/83518257/202909995-dcd1c82d-e452-49bc-96f2-e9a3e24fc401.png)
![imagen](https://user-images.githubusercontent.com/83518257/202910232-d3b34651-09a7-450e-bb83-7c83914d85e5.png)
![imagen](https://user-images.githubusercontent.com/83518257/202910420-eea1a8b6-ccb9-4cab-a881-f8b70b178029.png)
![imagen](https://user-images.githubusercontent.com/83518257/202910722-cbb75669-a59f-4aba-ab76-a267a76ca8ef.png)
![imagen](https://user-images.githubusercontent.com/83518257/202910883-0806e93f-f4c2-4590-8f8c-1550a47134a2.png)

**Reply by swarnava** [reply]
Thanks! How are you reproducing this? Which browser are you using?

**Reply by exitcas** [reply]
This screenshot was made on Librewolf v107.0-1

I've enabled the following list of addons at the moment:
- CORS Everywhere
- Ew, Mail!
- FrankerFaceZ
- Markdown Viewer
- PreMiD
- Simple Translate

Advanced against tracking is activated on "strict" mode

**Reply by amyegan** [reply]
Are you still having this problem @exitcas? If so, is there a specific page that I should visit to reproduce the error?
